moveit_demo_nodes/run_move_group/launch/run_move_group_interface.launch.py

```diff
@@ -1,55 +1,17 @@
-import os
-import yaml
 from launch import LaunchDescription
 from launch_ros.actions import Node
-from ament_index_python.packages import get_package_share_directory
-
-
-def load_file(package_name, file_path):
-    package_path = get_package_share_directory(package_name)
-    absolute_file_path = os.path.join(package_path, file_path)
-
-    try:
-        with open(absolute_file_path, "r") as file:
-            return file.read()
-    except EnvironmentError:  # parent of IOError, OSError *and* WindowsError where available
-        return None
-
-
-def load_yaml(package_name, file_path):
-    package_path = get_package_share_directory(package_name)
-    absolute_file_path = os.path.join(package_path, file_path)
-
-    try:
-        with open(absolute_file_path, "r") as file:
-            return yaml.safe_load(file)
-    except EnvironmentError:  # parent of IOError, OSError *and* WindowsError where available
-        return None
+from moveit_configs_utils.moveit_configs_builder import MoveItConfigsBuilder
 
 
 def generate_launch_description():
-    # planning_context
-    robot_description_config = load_file(
-        "moveit_resources_panda_description", "urdf/panda.urdf"
+    moveit_config = (
+        MoveItConfigsBuilder("moveit_resources_panda")
+        .robot_description(file_path="config/panda.urdf.xacro")
+        .robot_description_semantic()
+        .robot_description_kinematics()
+        .joint_limits()
+        .to_moveit_configs()
```

Comment:
I know you might not need all parameters everywhere, but why did you decide to specify every single parameter for each use? I think I might prefer
```
MoveItConfigsBuilder("moveit_resources_panda")
.robot_description(file_path="config/panda.urdf.xacro")
.to_moveit_configs()
```
to load all other parameters with their defaults.

Reply:
:+1: to this suggestion.  I think one of the large barriers to the usability of moveit is many things don't have defaults that should.  Here is one case where the default behavior (what parts of the moveit config to load) could be defined somewhere and the builder could then just override various parts of it.